cpp/testapps/cppWindowsRunnerGame/main.cpp

```diff
@@ -323,7 +328,8 @@ int main(int argc, char* argv[])
         Microsoft::Azure::Gaming::GSDK::start();
         Microsoft::Azure::Gaming::GSDK::registerShutdownCallback(&inShutdown);
         Microsoft::Azure::Gaming::GSDK::registerHealthCallback(&isHealthy);
-        Microsoft::Azure::Gaming::GSDK::registerMaintenanceCallback(&maintenanceScheduled);
+        //Microsoft::Azure::Gaming::GSDK::registerMaintenanceCallback(&maintenanceScheduled);
```

Comment:
should we remove the comment? doesn't hurt to test both methods I guess

Reply:
I left this commented mostly to give devs the example but the compiler options for the project promote the "deprecated" warning to an error. I commented out the same line in the C# example for similar reasons.

If we really want to check maintenance v1 callback, we can uncomment in the moment easily, but by default I think it makes more sense to leave commented out.